app/views/activities/_feed_entry.html.slim

```diff
@@ -1,10 +1,14 @@
 li class="activity #{activity.kind}"
   = link_to activity.source_url
-    = image_tag(activity.img_url || '/images/screenshot-missing.png')
-  = content_tag :h4, link_to(activity.title, activity.source_url, class: 'title')
+  	- if activity.img_url.present?
```

Comment:
It would seem that you are using tabs instead of spaces. Could you configure your editor(s) to use 2 spaces for indentation?